cmip6_downscaling/workflows/bcsd_flow.py

```diff
@@ -0,0 +1,112 @@
+import os
+
+from dask_kubernetes import KubeCluster, make_pod_spec
+from prefect import Flow, task
+from prefect.executors import DaskExecutor
+from prefect.run_configs import KubernetesRun
+from prefect.storage import Azure
+
+from cmip6_downscaling.methods.bcsd import (
+    fit_and_predict,
+    postprocess_bcsd,
+    prep_bcsd_inputs,
+    preprocess_bcsd,
+)
+from cmip6_downscaling.workflows.utils import make_flow_paths
+
+connection_string = os.environ.get("AZURE_STORAGE_CONNECTION_STRING")
+
+run_hyperparameters = {
+    "FLOW_NAME": "BCSD_testing",
+    "GCM": "MIROC6",
+    "SCENARIO": "ssp370",
+    "TRAIN_PERIOD_START": "1990",
+    "TRAIN_PERIOD_END": "1990",
+    "PREDICT_PERIOD_START": "2090",
+    "PREDICT_PERIOD_END": "2090",
+    "VARIABLE": "tasmax",
+    "OBS": "ERA5",
+}
+
+storage = Azure("prefect")
```

Comment:
Maybe we could move out the config stuff into some shared file and import it?
Maybe this is a future PR..